src/controllers/poll.js

```diff
@@ -12,19 +10,22 @@ export const createPoll = async (req, res) => {
     res.status(201).json({ poll });
   } catch (error) {
     res.status(409).json({ message: error.message });
```

Comment:
is 409 the right error code?

Reply:
yes, as we are updating the record, it might be a conflict in data consistency. so  409 good candidate here